build.ps1

```diff
@@ -61,9 +60,15 @@ else {
         ExecSafe { & powershell $DotNetInstallFile -InstallDir $DotNetDirectory -Version $DotNetVersion -NoPath }
     }
     $env:DOTNET_EXE = "$DotNetDirectory\dotnet.exe"
+    $env:PATH = "$DotNetDirectory;$env:PATH"
 }
 
 Write-Output "Microsoft (R) .NET SDK version $(& $env:DOTNET_EXE --version)"
 
+if (Test-Path env:NUKE_ENTERPRISE_TOKEN) {
```

Comment:
curious about this... but can't find much on the internet...

Reply:
Taken while updating based on: https://github.com/nuke-build/nuke/tree/8.0.0/source/Nuke.GlobalTool/templates

Introduced in: https://github.com/nuke-build/nuke/commit/74a24349c8e94a78d0a9127f4a3adb9d7c001271. I do not know this feature :)